tomls/omnibus-arbitrum-sepolia/rewards/sc.toml

```diff
@@ -0,0 +1,65 @@
+[provision.RewardsDistributor_SpartanCouncilPool]
```

Comment:
This one needs all the new e2e tests added or existing ones updated

Reply:
i will keep the old tests (since the old RDs are still in  play for a bit longer) and add new one as you suggest :+1: